product_docs/docs/migration_portal/4/01_mp_release_notes/mp_4.11.0_rel_notes.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Migration Portal 4.11.0 release notes"
+navTitle: Version 4.11.0
+---
+
+Released: 20 Aug 2024
+
+New features, enhancements, bug fixes, and other changes in Migration Portal 4.11.0 include the following:
+
+| Type              | Description                                                                                                      |
+|-------------------|------------------------------------------------------------------------------------------------------------------|
+| Enhancement       | Updated the styling for logos, fonts and layouts for an improved look and feel.                                  |
+| Security&nbsp;fix | Updated the software libraries and implemented other security improvements.                                      |
```

Comment:
I recommend changing "Updated the software libraries and ..." to "Updated software libraries and ..."